test-cases/performance/perf-regression.100threads.30M-keys.yaml

```diff
@@ -13,12 +13,25 @@ n_db_nodes: 3
 n_loaders: 4
 n_monitor_nodes: 1
 
+#AWS
 instance_type_db: 'i3.2xlarge'
 instance_type_loader: 'c4.2xlarge'
 instance_type_monitor: 't3.small'
-
-user_prefix: 'perf-regression'
 space_node_threshold: 644245094
+#------
+#GCE
+gce_instance_type_db: 'n2-highmem-8'
+gce_root_disk_type_db: 'pd-ssd'
+gce_n_local_ssd_disk_db: 8
+gce_instance_type_loader: 'c3-highcpu-8'
+gce_root_disk_type_loader: 'pd-ssd'
+root_disk_size_loader: 50
+gce_instance_type_monitor: 'n2-highmem-8'
+#+ rewrite var from  perf-regression-throughput-config-gce-modifications.yaml
+#use_preinstalled_scylla: true
+#space_node_threshold: 0
```

Comment:
can be removed.

Reply:
do you mean we can leave space_node_threshold: 644245094 git GCE?

Reply:
this param is only relevant for longevity to start a nemesis.

Reply:
removed